vignettes/package.Rmd

```diff
@@ -103,7 +100,7 @@ extendr-api = '*'
 The crate name is the same name as the R package's name by default. You can change this,
 but it might be a bit cumbersome to tweak other files accordingly, so we recommend leaving this.
 
-To try the dev version of the extendr, you can modify the last line to
+You will probably want to specify a concrete extendr version, for example `extendr-api = '0.2'`. To try the development version of the extendr, you can modify the last line to read
```

Comment:
I was wondering if I should write this. As long as the package author commits `Cargo.lock`, the build should be reproducible. Of course one might want to specify the older version than the latest released version, but it's probably a rare case.

Reply:
I think it's good practice to specify a version instead of relying only on `Cargo.lock`. Also, I wanted to clarify that the `'*'` isn't there for some important reason.

Reply:
I see, sounds good.